lib/external_service_new/zendesk_downloader.rb

```diff
@@ -0,0 +1,51 @@
+module ExternalServiceNew
+	class ZendeskDownloader
+		def initialize(dispatcher:, logger:nil)
+			@dispatcher = dispatcher
+			@logger			= logger || respond_to?(:logger) ? logger : nil
+		end
+
+		def each_list(&block)
+			return to_enum(:each_list) unless block
+			[[
+				dispatcher.site_url,
+				"#{dispatcher.site_url}'s customers"
+			]].each(&block)
+		end
+
+		def each_email(id: nil, &block)
```

Comment:
I saw it like this in [salesforce_downloader.rb](https://github.com/plaftmp/wafflehouse/blob/plaf/lib/external_service_new/salesforce_downloader.rb#L39)
It's maybe to unify all the downloader